js/src/elements/form-behavior.ts

```diff
@@ -65,7 +65,6 @@ export class FormBehaviorElement extends HTMLElement {
 
 		form.setAttribute('novalidate', '');
 		const errorSummaryContainer = document.createElement('div');
-		errorSummaryContainer.classList.add('margin-bottom-sm', 'background-color-danger-light');
```

Comment:
Here's the correct preview link https://design.docs.microsoft.com/pulls/454/components/form.html

Until we implement banners in Atlas, let's keep the background color class for now.

Reply:
Yeah, previewed it and realized we need it. Moved it to the `alert` class.

Reply:
I'm guessing alot fo this will change with banners so trying to do the minumum adjust here.

Reply:
Correct, as the current `.alert` in Docs doesn't quite match the new banner designs anyway.